scripts/runWithVars.sh

```diff
@@ -26,10 +26,10 @@ export REACT_APP_BUILD_NUMBER=${TRAVIS_BUILD_NUMBER:-"0"}
 export REACT_APP_VERSION="$VERSION_MAYOR.$VERSION_MINOR.$REACT_APP_BUILD_NUMBER"
 export REACT_APP_TITLE=$npm_package_title
 
-# TODO:
-# need to copy the pollifill from the library
-#  node_modules/web-vitals/dist/polyfill.js
-# sed  's/<script id="web-vitals">/,/<\/script>/\<\/bla>/g;' public/index.html > public/index.html
+# Copy web-vitals polyfill script from node_modules/web-vitals/dist/polyfill.js to public/index.html
+SCRIPT=`cat node_modules/web-vitals/dist/polyfill.js`
+SCRIPT=$(echo "$SCRIPT" | sed -e 's/&/\\\&/g' )
+sed -i "" "s#<script id=\"web-vitals\">#<script id=\"web-vitals\">$SCRIPT#" public/index.html
```

Comment:
need to test it in github runs (or in Linux )

Reply:
we don't have a github action running so probably need to get it set up, or have to set it in the Travis CI.

Reply:
I've meant CI

Reply:
It has passed in CI.
https://travis-ci.org/github/Geovation/photos/builds/773785185